### ISSUE TITLE
SystemCalibrationXds: receptor_label

### ISSUE THREAD

**Issue by kettenis** [casangi/xradio#338]
Specified as `int64` in the schema, where I think this should be `<U0` like in the GainCurveXds with `polarization_type` added.

**Comment by FedeMPouzols** [comment]
Yes, string type + `polarization_type` is what we have in `AntennaXds`, `GainCurveXds` and `PhaseCalibrationXds`, but for `SystemCalibrationXds` it remained as int in the spreadsheet. If there was not particular reason for this, we should align them.

This is by the way one of the places where #294 should be considered.

**Comment by Jan-Willem** [comment]
Yes, it should be a string. Will fix.

**Comment by FedeMPouzols** [comment]
I've opened a PR to fix this: 347. Note that this also makes the `polarization_type` coordinates of `gain_curve_xds` and `phase_calibration_xds` mandatory. They were optional but I don't see that in the spreadsheet, or any other reason to have them as optional. I suspect it was a leftover from earlier schema versions. The converter was already populating them in any case.

**Comment by FedeMPouzols** [comment]
This was done in PR #347.
